testsuite/features/secondary/srv_timezone_feature

```diff
@@ -0,0 +1,140 @@
+# Copyright (c) 2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+#
+#  1) create a user and assing him a timezone different than the server's timezone
+#  2) test that the popups in some scheduling actions appear in users prefered timezone 
+#  3) some scheduler tests based on previous bugs are unavoidable
```

Comment:
Ideally, the description or acceptance criteria for the feature goes just below the keyword "Feature:"

Reply:
Yes, please describe what the software feature is supposed to do, from an end-user point of view. Don't describe the scenarios below.